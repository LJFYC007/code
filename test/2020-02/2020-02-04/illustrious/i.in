5
5
50
500
1000
500000
