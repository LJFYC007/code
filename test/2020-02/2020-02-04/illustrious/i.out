1 2 2 3 3 4 4 4 5 5 5 6 6 6 6 7 7 7 7 8 8 8 8 9 9 9 9 9 10 10 
1 3 5 8 11 15 19 23 28 33 38 44 50 56 62 69 76 83 90 98 106 114 122 131 140 149 158 167 177 187 
50
40441
62542986
573052080
698264366
